src/ast.rs

```diff
@@ -0,0 +1,92 @@
+pub struct Item {
+    pub is_pub: Boolean,
+    pub ident: String,
+    pub kind: ItemKind,
+}
+
+pub enum ItemKind {
+    Fn(Fn),
+    Struct,
+    Enum,
+    Mod,
+    Use,
+    Trait,
+    TyAlias,
+}
+
+pub struct Fn {
+    pub generics: Generics,
```

Comment:
Is it ok not include defaultness? `Fn` in rustc ast does.

Reply:
I thought that there is no way to change that defaultness when I saw rust syntax docs. Actually, I couldn't understand the PR of function defaultness feature deeply. It looks like making final function which cannot override more, but figuring out how to make it final or what situation needs this in rust may need more time. How about change this when needs occurs while parsing?